llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -1932,22 +1932,6 @@ void DwarfDebug::collectEntityInfo(DwarfCompileUnit &TheCU,
     // Finalize the entry by lowering it into a DWARF bytestream.
     for (auto &Entry : Entries)
       Entry.finalize(*Asm, List, BT, TheCU);
-    List.finalize();
```

Comment:
Can you give some rationale behind this removal?

Reply:
This change comes from the original code https://github.com/llvm/llvm-project/blob/release/13.x/llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp#:~:text=for%20(auto%20%26Entry,List%2C%20BT%2C%20TheCU)%3B
I pushed as-it-is

Reply:
Paul commented offline that this change comes from release_12x, I see that the issue was that the base code for release/13.x is different and when I applied the commit for this piece of code the build was breaking, hence I remove the code. What I will do now is to pick up the commit again and do the required changes for release_13x be compatible with it. I will commit-push again once is ready.